BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -1822,6 +1867,9 @@ void EndBoost()
                     if (gEmitter.Current == null) continue;
                     gEmitter.Current.emit = false;
                 }
+            
+            if (useFuel) ordinanceMass = -boosterMass;
+            
```

Comment:
boosterMass, as seen just a few lines down, has already been defined as the mass of the entire booster assembly - casing, nozzle, fuel, etc. - in the few missile configs already making use of it (see e.g. the stock RBS15-17). Might be worth changing ``sustainerMass``/``boosterMass`` to something like ``fuelMass``/``boosterFuelMass``, to maintain compatibility with existing configs.